playground/webpack.config.prod.js

```diff
@@ -3,7 +3,9 @@ const createWebpackConfigForProduction = require('@commercetools-frontend/mc-scr
 
 const distPath = path.resolve(__dirname, 'dist');
 const entryPoint = path.resolve(__dirname, 'src/index.js');
-const sourceFolders = [path.resolve(__dirname, 'src')];
+const sourceFolders = [path.resolve(__dirname, 'src')].concat[
+  'node_modules/omit-empty'
```

Comment:
this dependency ships untranspiled  code.

Reply:
Can you explain a bit more what the actual problem is? 🤔

Reply:
I updated the description

Reply:
The problem is the MC doesn't work in IE11 because our bundle contains ES6 features like arrow functions.